packages/dev-scripts/bin/sf-nuts.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const shell = require('../utils/shelljs');
+
+const packageRoot = require('../utils/package-path');
+const { resolveConfig } = require('../utils/sf-config');
+
+const nyc = require.resolve('nyc/bin/nyc');
+
+const config = resolveConfig(packageRoot);
+const testConfig = config.test || {};
+const includes = testConfig.testsPath || '**/*.nut.ts';
+
+const command = `${nyc} mocha "${includes} --timeout 600000 --parallel"`;
```

Comment:
can this also have  `--slow 4500` (otherwise they're all read and kinda meaningless).  4500 gives me this

![Screen Shot 2021-02-19 at 1 45 18 PM](https://user-images.githubusercontent.com/4261788/108553724-b7e94880-72b8-11eb-81e4-b2b067086830.png)

Reply:
I think they are meaningless for nuts anyway but I don't mind adding it.